scripts/cloud-check-control/cloudAtlasCheckControl.py

```diff
@@ -231,6 +244,55 @@ def sync(self):
         if self.terminate:
             self.terminate_instance()
 
+    def challenge(self):
+        logger.info("Creating map roulette challenge.")
+
+        # sync the country folders to the local directory
+        for c in list(self.countries.split(",")):
+            logger.info(
+                "syncing {}/flag/{} to {}/flag/{}".format(
+                    self.s3OutFolder, c, self.atlasOutDir, c
+                )
+            )
+            if self.ssh_cmd(
+                "aws s3 sync --only-show-errors s3://{0}/flag/{1} {2}/flag/{1}".format(
+                    self.s3OutFolder, c, self.atlasOutDir
+                )
+            ):
+                finish("Failed to sync S3://{}/{}".format(self.s3OutFolder, c), -1)
+        if self.ssh_cmd(
+            "aws s3 cp s3://{} {}".format(
+                self.atlasConfig, os.path.join(self.homeDir, self.atlasConfig)
+            )
+        ):
+            finish("Failed to copy config S3://{}".format(self.atlasConfig), -1)
+
+        # push output to s3
+        cmd = "aws s3 sync --only-show-errors --exclude *.crc {} s3://{} ".format(
+            self.atlasOutDir, self.s3OutFolder
+        )
+        if self.ssh_cmd(cmd):
+            finish("Unable to sync with S3", -1)
+
+        cmd = (
+            "java -cp atlas-checks/build/libs/atlas-checks-*-SNAPSHOT-shaded.jar"
```

Comment:
Could we make the jar location configurable?

Reply:
Sure.